src/main/scala/com/typesafe/sbt/pom/PomReaderPlugin.scala

```diff
@@ -0,0 +1,15 @@
+package com.typesafe.sbt.pom
+
+import sbt._
+import sbt.plugins.JvmPlugin
+
+/** Plugin definition for Maven POM reader. */
+object PomReaderPlugin extends AutoPlugin {
+  object autoImport extends SbtPomKeys {
+    @deprecated("This is an AutoPlugin; settings are automatically added. Using this value is no longer required", "1.1")
+    lazy val useMavenPom = projectSettings
```

Comment:
@jsuereth For backward compatibility with pre-AutoPlugin usages, is this the right way communicating to users that a initialization call is no longer needed?